code/modules/jobs/job_types/scientist.dm

```diff
@@ -9,6 +9,8 @@
 	supervisors = "the research director"
 	selection_color = "#ffeeff"
 	chat_color = "#C772C7"
+	minimal_player_age = 7
+	exp_requirements = 600
```

Comment:
10 hours is a pretty long time, this is very limiting to new players almost pointlessly, since most griefers will just break in or get promoted anyway.

Reply:
Having to break in really makes all the difference in identifying them as a griefer.

Given how complex most of science is, and especially due to circuits, I'm pretty firm on my belief of this one too.

Reply:
Science is really fun for a lot of new people _because_ of that complexity though. How much are we willing to degrade the experience of new players for grief protection

Reply:
That's such an ass-backwards statement you just made that I'm not really even sure exactly how to reply; Science is really fun for it's complexity, but complexity is *confusing* for new players who barely have basic mechanics down. complex jobs are not accessible at all for truly new players until they have a fair bit of general experience with the game under their belt.

Of all the jobs reqs adjusted, I am not budging on Scientist (notable this does not include roboticist), all of Engineering and all of Security needing at least ten hours of play - not just because of grief, but because the jobs require a certain level of general experience before it's even feasible to perform the job up to expectations, let alone understand the consequences of your actions when you've done something wrong. 

This is the hill this PR will die on if it must.

Reply:
You're presuming the players aren't coming from different servers. This dissuades anyone who is trying to migrate from /tg/ or similar.

Reply:
And it's something admins can override with the click of a button upon request and briefly verifying they aren't known for grief.